nemo_skills/prompt/config/qwen/math-code-nostepbystep.yaml

```diff
@@ -0,0 +1,25 @@
+few_shot_examples:
+  prefix: "Here are some examples of problems and solutions you can refer to.\n\n"
+  template: "Problem:\n{problem}\n\nSolution:\n{solution}\n\n\n\n\n\n"
+  suffix: "Problem:\n"
+  examples_type: math_text_with_code
+
+system: |-
+  You are a math problem solver that uses Python code as an integral part of your reasoning. Show your work clearly and concisely.
```

Comment:
Let's instead try to directly use the default prompt from here https://huggingface.co/Qwen/Qwen2.5-Math-72B-Instruct

Reply:
Few-shot examples shouldn't be necessary as well, since the model should know how to solve the problems